force-app/main/default/lwc/geGatewaySelectWidget/geGatewaySelectWidget.js

```diff
@@ -188,15 +202,15 @@ export default class GeGatewaySelectWidget extends LightningElement {
     }
 
     async handleInitialGatewaySelection(elevateSettings) {
-        if (isEmpty(elevateSettings) || isEmpty(elevateSettings.uniqueKey)) {
+        if (isEmpty(elevateSettings) || this._widgetMode === GATEWAY_MANAGEMENT_MODE) {
```

Comment:
There are enough instances of `this._widgetMode === GATEWAY_MANAGEMENT_MODE` here that I think a helper method would be cleaner. Could be more descriptive with it as well, something like, `onSettingsPage`, vs `onBatchSetup`, just to differentiate the modes of this component.